ci/prometheus-down.sh

```diff
@@ -20,7 +20,10 @@ set -u
 : ${PROMETHEUSHOST:="0.prometheus.production-monitoring.prometheus-production.toolingbosh"}
 : ${ALERTMANAGERHOST:="0.alertmanager.production-monitoring.prometheus-production.toolingbosh"}
 : ${QUERIES:="
-  push_time_seconds
+  push_time_seconds%7Bjob%3D\"concourse_has_auth\"%7D
+  push_time_seconds%7Bjob%3D\"boshunknowninstance-lastcheck\"%7D
```

Comment:
This metric is redundant as of push gateway 0.5 and isn't emitted any more.

Reply:
Ah.  I did some interpolation with names, and thought I'd checked that the data was there, but I guess I didn't do this.  Thanks!